packages/@expo/cli/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 ### 🐛 Bug fixes
 
+- Stop corrupting inner `.tar.gz` files when decompressing tarballs. ([#26741](https://github.com/expo/expo/pull/26741) by [@shirakaba](https://github.com/shirakaba))
```

Comment:
```suggestion
- Mark compressed `.gz` files as binary to avoid corruption when unpacking with `expo prebuild --template`. ([#26741](https://github.com/expo/expo/pull/26741) by [@shirakaba](https://github.com/shirakaba))
```